apps/mobile/src/navigation/MMBStack.js

```diff
@@ -72,6 +73,10 @@ const StackNavigator: NavigationNavigator<
         title: 'Trip Services',
       },
     },
+    [Routes.PASSENGER_DETAIL]: {
+      screen: PassengerDetailScreen,
+      navigationOptions: {
+        title: 'Passengers',
```

Comment:
There are missing a couple of closing brackets here

Reply:
oh was accidentally deleted while merging